cmd/minikube/cmd/config/config.go

```diff
@@ -213,6 +218,79 @@ func configurableFields() string {
 	return strings.Join(fields, "\n")
 }
 
+// AddToConfigArray adds entries to an array in the config file
+func AddToConfigArray(name string, images []string) error {
+	s, err := findSetting(name)
+	if err != nil {
+		return err
+	}
+
+	// Set the values
+	configFile, err := config.ReadConfig()
+	if err != nil {
+		return err
+	}
+	values := configFile[name]
+
+	// Add images to currently existing values in config file
+	if values != nil {
+		for _, v := range values.([]interface{}) {
+			images = append(images, v.(string))
+		}
+	}
+
+	err = s.setArray(configFile, name, images)
+	if err != nil {
+		return err
+	}
+
+	// Write the values
+	return WriteConfig(configFile)
+}
+
+// DeleteFromConfigArray deletes entries in an array in the config file
+func DeleteFromConfigArray(name string, images []string) error {
+	s, err := findSetting(name)
+	if err != nil {
+		return err
+	}
+	// Set the values
+	configFile, err := config.ReadConfig()
+	if err != nil {
+		return err
+	}
+	values := configFile[name]
```

Comment:
i think you want a `val, ok := configFile[name]`.  `ok` is a boolean that is true if the key exists in the map, and false if not.